package.json

```
@@ -8,10 +8,10 @@
   "repository": "https://github.com/AwesomeDocs/AwesomeDocs",
   "scripts": {
     "build": "./node_modules/.bin/gatsby build",
-    "lint": "./node_modules/.bin/tslint -p tsconfig.json",
     "start": "./node_modules/.bin/gatsby develop",
     "test": "yarn lint && yarn transpile",
-    "transpile": "./node_modules/.bin/tsc"
+    "transpile": "./node_modules/.bin/tsc",
+    "lint": "yarn eslint . --ext .js --ext .ts --fix"
```

This won't lint any of the files, use this instead:
```suggestion
    "lint": "./node_modules/.bin/eslint . --ext .js,.jsx,.ts,.tsx"
```